corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -260,10 +260,14 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.geocoderItemCallback = function (item) {
             self.rawAnswer(item.place_name);
             self.editing = false;
+            var broadcastObj = Utils.getBroadcastObject(item);
             self.broadcastTopics.forEach(function (broadcastTopic) {
-                var broadcastObj = Utils.getBroadcastObject(item);
                 question.parentPubSub.notifySubscribers(broadcastObj, broadcastTopic);
             });
+            question.answer(JSON.stringify(broadcastObj));
+            if (!question.answer()) {
+                question.answer("");
```

Comment:
There's a `NO_ANSWER` constant that would be good to use here for clarity.

Reply:
I looked at that, but I was thinking `answer` needed to be an empty string as opposed to `null`.